tests/bazel_run_tests.sh

```diff
@@ -367,8 +379,10 @@ TESTS=(
   "test_transitive_dependency_with_type_of_pom"
   "test_when_both_pom_and_jar_artifact_are_available_jar_artifact_is_present"
   "test_when_both_pom_and_jar_artifact_are_dependencies_jar_artifact_is_present"
-  # "test_gradle_metadata_is_resolved_correctly_for_aar_artifact"
+  "test_gradle_metadata_is_resolved_correctly_for_aar_artifact"
```

Comment:
For context about this change see [here](https://github.com/bazel-contrib/rules_jvm_external/pull/1395/files#r2264116068)